src/includes/getting-started-primer/apple.mdx

```diff
@@ -20,3 +20,29 @@
 - [Release Health](/product/releases/health/) tracks crash free users and sessions
 - [Attachments](/platforms/apple/enriching-events/attachments/) enrich your event by storing additional files, such as config or log files
 - [User Feedback](/platforms/apple/enriching-events/user-feedback/) provides the ability to collect user information when an event occurs
+
+**Features using Swizzling:**
+
+The Cocoa SDK uses [swizzling](https://nshipster.com/method-swizzling/) to provide some features out of the box without boilerplate code. The following features use swizzling:
+
+
+
+Since Cocoa <= 7.5.0, you can opt out of swizzling using options. When disabling swizzling, the SDK disables the features above:
```

Comment:
Just making the same changes here.

```suggestion
Since Cocoa 7.5.0, you can opt out of swizzling using options. When you disable swizzling, the SDK disables the features above:
```